packages/browser/src/plugins/ajs-destination/index.ts

```diff
@@ -192,7 +192,7 @@ export class LegacyDestination implements InternalPluginWithAddMiddleware {
     return (
       // page events can't be buffered because of destinations that automatically add page views
       ctx.event.type !== 'page' &&
-      (isOffline() || this._ready === false || this._initialized === false)
+      (isOffline() || this._ready !== true || this._initialized !== true)
```

Comment:
Nit: Couldn't we just get rid of the explicit false and do if (!this.ready) etc?